modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view.jsp

```diff
@@ -101,6 +101,7 @@ DLViewDisplayContext dlViewDisplayContext = new DLViewDisplayContext(dlAdminDisp
 					"viewFileEntryURL", dlViewDisplayContext.getViewFileEntryURL()
 				).build()
 			%>'
+			disableIncompatibleActions="<%= true %>"
```

Comment:
This attribute is on the level of the whole management toolbar. Shouldn't this be on the level of individual actions? For example, in D&M, devs would want the `move` action to be enabled and `permissions` to be disabled.

Reply:
Okay, I wanted to present this solution, as it's more generic, so we can see if it's enough or if I should make it more complex.

Do you have an idea how the API should be exposed @markocikos @dsanz ?
- Does the user whitelist which actions should be disabled if incompatible document types are selected?
- Does the user blacklist actions? 
- Something else?

Reply:
IIUC, the behavior we want is:
- All bulk actions are enabled by default. It doesn't matter if single or multiple types are selected in UI.
- Devs can set a flag in action definition. If that flag is set to true, action will be disabled when multiple document types are selected in UI.

Reply:
Should the API even care about incompatible document types if it's going to be like you're implying?
> For example, in D&M, devs would want the move action to be enabled and permissions to be disabled.

In that case it's not related to the title of the task:
> Allow developers to disable bulk actions when selecting elements of different types

Reply:
We still need to check if multiple types are selected. Only then should the action be disabled.

I am not sure if you mean anything different with 'incompatible' types. There is no compatibility mapping, they are all incompatible of they are of different types.

Reply:
Yeah okay 👍 I'll cook something up

<img src="https://media3.giphy.com/media/cIyHhlLkVGSL3Cnc9o/giphy.gif"/>